modules/@apostrophecms/schema/ui/apos/components/AposInputWrapper.vue

```diff
@@ -141,6 +142,7 @@ export default {
     }
   },
   mounted: function () {
+    console.log('mount time:', JSON.stringify(this.originalDoc, null, '  '));
```

Comment:
Works. It looks like by the time fields are mounted `originalDoc.ref` has already become an object, but if it hadn't we could have used `watch` to wait for that.